addons/web/static/src/views/view_button/multi_record_view_button.js

```diff
@@ -24,4 +24,9 @@ export class MultiRecordViewButton extends ViewButton {
     }
 }
 
-MultiRecordViewButton.props = [...ViewButton.props, "list", "domain"];
+MultiRecordViewButton.props = [...ViewButton.props, "list", "domain", "shouldTakeSelectedRecords?"];
+
+MultiRecordViewButton.defaultProps = {
+    ...ViewButton.defaultProps,
+    shouldTakeSelectedRecords: true,
+};
```

Comment:
props and defaultProps can be defined as static attributes in the class of the component. 🙂